tests/python/mints14/input.py

```diff
@@ -0,0 +1,74 @@
+                   1.15152678e-02,  2.05265305e-01,  2.83124071e-01,  3.85914212e-02,
+                   1.81738886e-02,  1.07752887e-02,  1.77818258e-02,  3.68574105e-02,
+                   2.25974306e-02,  1.35771735e-02,  8.87906889e-03,  5.10066439e-03,
+                   1.12829223e-02,  1.12265555e-02,  8.83213803e-03,  6.63860586e-03,
+                   2.19102261e-03,  4.93006854e-03,  5.89615996e-03,  5.53808484e-03,
+                   4.71886332e-03, -5.17239329e-02,  3.13726711e-02,  2.75355388e-02,
+                   1.65919492e-02,  1.03990059e-02, -3.35874097e-03,  4.15222788e-02,
+                   2.56191309e-02,  1.52660739e-02,  9.76796362e-03,  6.32834557e-03,
+                   2.04241605e-02,  1.70973880e-02,  1.17455466e-02,  8.13966708e-03,
+                   3.38098123e-03,  8.74425646e-03,  9.48996966e-03,  7.93026739e-03,
+                   6.17969093e-03,  1.74806282e-03,  4.26579485e-03,  5.29784951e-03,
+                   5.12333890e-03,  4.45846711e-03, -1.57567665e-02,  5.30695677e-03,
+                   1.23293022e-02,  1.06586884e-02,  7.93900611e-03, -7.73802957e-03,
+                   7.26178250e-03,  1.17147293e-02,  9.98332008e-03,  7.52680394e-03,
+                  -4.70874042e-04,  7.07678716e-03,  9.28214077e-03,  8.16156593e-03,
+                   6.44348860e-03,  1.00781695e-03,  4.75096863e-03,  6.24946368e-03,
+                   5.97476305e-03,  5.07885832e-03,  9.07850148e-04,  2.91174728e-03,
+                   3.98405846e-03,  4.14293807e-03,  3.80513995e-03, -5.01856357e-03,
+                   1.66005706e-03,  5.58988126e-03,  6.25608633e-03,  5.51718510e-03,
+                  -3.54538877e-03,  2.13874136e-03,  5.42317564e-03,  5.96499476e-03,
+                   5.28687977e-03, -1.26041313e-03,  2.57086533e-03,  4.76216983e-03,
+                   5.15904477e-03,  4.66727597e-03, -6.98718627e-05,  2.29543121e-03,
+                   3.72078488e-03,  4.09219098e-03,  3.84409282e-03,  2.86099206e-04,
+                   1.74337099e-03,  2.70404545e-03,  3.07010847e-03,  3.01768294e-03,
+                  -2.17493011e-03,  7.08011269e-04,  2.82422496e-03,  3.69989978e-03,
+                   3.71459579e-03, -1.77325890e-03,  8.55302891e-04,  2.77277336e-03,
+                   3.57535348e-03,  3.59320794e-03, -9.75050076e-04,  1.08091443e-03,
+                   2.57027134e-03,  3.22234880e-03,  3.25925100e-03, -3.45599629e-04,
+                   1.12615341e-03,  2.20372521e-03,  2.71976417e-03,  2.79365593e-03,
+                  -1.83114358e-05,  1.00066735e-03,  1.76948007e-03,  2.18338059e-03,
+                   2.29358229e-03]
+
+# Comparison
+for i in range(0, len(points)):
+    psi4.compare_values(esps_1threads[i],esps_4threads[i],3,"Reference value for ESP calculation, 1 thread vs. 4 threads.")
+    psi4.compare_values(esps_1threads[i],reference_esps[i],3,"Reference value for ESP calculation, 1 thread vs. reference calculation.")
+
```

Comment:
It's cleaner with array tools:
`psi4.compare_arrays(esps_1threads,esps_4threads,...`

Reply:
Thank you, didn't know about this function.
I added a commit.